Sources/SwiftParser/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ add_swift_host_library(SwiftParser
   Specifiers.swift
   Statements.swift
   StringLiterals.swift
+  StringLiteralRepresentedLiteralValue.swift
```

Comment:
Oh this is now unrelated to this PR. I wanted to use this feature (`representedLiteralValue `) in `swift` repo test cases, but ended up with not using it because the compiler doesn't support `@freestanding(declaration, names: arbitrary)` macros.

But this was missing anyway.